math/src/stableswap/types.rs

```diff
@@ -0,0 +1,29 @@
+use crate::types::Balance;
+use num_traits::Zero;
+
+#[derive(Debug, Clone, Copy)]
+pub struct AssetReserve {
+	pub amount: Balance,
+	pub decimals: u8,
+}
```

Comment:
How will you pass this over the WASM boundary?

Reply:
it is done using json format, I could not come up with better way.  and it is deserialized from that. so FE already uses this. seems to work ok